books/free-programming-books-langs.md

```diff
@@ -394,6 +395,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Visual Basic Essentials](http://www.techotopia.com/index.php/Visual_Basic_Essentials)
 
 
+### BeanShell
+
+* [Beanshell Simple Java Scripting Manual](http://www.beanshell.org/manual/bshmanual.pdf) (PDF)
+* [BeanShell User's Manual](http://www.beanshell.org/manual/bshmanual.html) (HTML)
+* [Using Beanshell for Beginners](https://www.blazemeter.com/blog/using-beanshell-beginners-no-java-knowledge-required) - Bob Meliev
```

Comment:
[CONTRIBUTING - Genres we don't list](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#genres-we-dont-list): blog post